docs/sample_config.yaml

```diff
@@ -907,26 +907,26 @@ uploads_path: "DATADIR/uploads"
 # *trusted* identity servers. Note that this allows the configured identity server to
 # reset passwords for accounts!
 #
-# Also be aware that if email is not set to a domain, and SMTP options have not been
-# configured in the email config block, registration and resetting user passwords via
+# Be aware that if `email` is not set, and SMTP options have not been
+# configured in the email config block, registration and user password resets via
 # email will be globally disabled.
 #
 # Additionally, if msisdn is not set to a domain, registration and password resets via
 # msisdn will be disabled regardless. This is due to Synapse currently not supporting
 # any method of sending SMS messages on its own.
 #
 # To enable using an identity server for operations regarding a particular third-party
-# identifier type, set the value to the reachable domain name of that identity server,
-# including protocol definition, e.g:
-#  email: "https://matrix.org"  # Let matrix.org handle sending emails for my users
-#  msisdn: "http://localhost:8090"  # Delegate contacting SMS numbers to this process
+# identifier type, set the value to the URL of that identity server, e.g:
+#  email: https://matrix.org  # Let matrix.org handle sending emails for my users
```

Comment:
rather than repeating the examples, I would say "as shown in the examples below"